src/entity/Result.js

```diff
@@ -20,7 +20,10 @@ class Result extends Entity {
      * @returns {Array.|null}
      */
     getItems() {
-        return typeof this.data[this.dataKey] !== 'undefined' ? this.data[this.dataKey] : [];
+        return typeof this.data[this.dataKey] !== 'undefined'
+            ? this.data[this.dataKey].map(item => this.createItem(item))
+            : []
+            ;
```

Comment:
1) identation
2) `[Symbol.iterator]` will fail as `getItems()` result is changed